.netlify/build.sh

```diff
@@ -23,6 +23,10 @@ else
     echo "<meta http-equiv=refresh content=0;url=v${PYO3_VERSION}/>" > netlify_build/index.html
 fi
 
+## Generate towncrier release notes
+
+pip install towncrier
+towncrier build --yes --version Unreleased --date TBC
```

Comment:
When doing a new release, do we need to update this version argument?

Reply:
I think not for the Netlify build which will always contain the latest not-yet-released changes.

We will however have to run `towncrier build` when creating a release which will remove the news fragments from version control after having merged their contents into the changelog.

Reply:
Correct, the release step to update the real CHANGELOG is manual.

I'm going to take a moment in the next week or two to make some notes on how to do releases, even though I hope to continue to be doing them for the foreseeable future!